bundles/org.openhab.ui/web/src/components/app.vue

```diff
@@ -750,10 +703,7 @@ export default {
       if (window) {
         window.addEventListener('keydown', this.keyDown)
       }
-
-      if (localStorage.getItem('openhab.ui:webaudio.enable') === 'enabled') {
-        this.startEventSource()
-      }
+      this.startEventSource()
```

Comment:
Let‘s add an empty line for readability.

```suggestion

      this.startEventSource()
```